CMakeLists.txt

```diff
@@ -55,4 +57,3 @@ set(GC_LIB_LINKED_LIBS
 )
 target_link_libraries(graph_compiler PRIVATE ${GC_LIB_LINKED_LIBS})
 
-add_subdirectory(test)
```

Comment:
What is this for?

Reply:
just moved it up. This folder contains gtests and lits.